graphql/schema/types/project_settings.graphql

```diff
@@ -11,6 +11,11 @@ enum ProjectSettingsSection {
   PLUGINS
   CONTAINERS
   VIEWS_AND_FILTERS
+
+  # GitHub sections.
+  GITHUB_AND_COMMIT_QUEUE
+  GITHUB_APP_SETTINGS
+  GITHUB_PERMISSION_GROUPS
```

Comment:
[nit] Thoughts on naming this `GITHUB_PERMISSIONS` in case permission management is expanded in the future?